lang.toml

```diff
@@ -20,32 +20,32 @@ organizer = { name = "t-lang", email = "lang@rust-lang.org" }
 recurrence_rules = [ { frequency = "weekly" } ]
 
 [[events]]
-uid = "9fa5f574898e6a2417d9ac9b20b5294507c15e2a"
-title = "Lang Team Planning Meeting"
-description = "In this meeting, we plan the design meetings for the month."
+uid = "2be23d34f40821a2793cf8bf2db5b1bcbfe620f3"
+title = "Lang Team Design Meeting"
+description = "https://github.com/orgs/rust-lang/projects/31/views/10"
 location = "https://meet.jit.si/ferris-rules"
 last_modified_on = "2024-02-29T00:00:00.00Z"
 start = { date = "2024-03-06T12:30:00.00", timezone = "America/New_York" }
 end = { date = "2024-03-06T14:00:00.00", timezone = "America/New_York" }
 status = "confirmed"
 transparency = "opaque"
 organizer = { name = "t-lang", email = "lang@rust-lang.org" }
-recurrence_rules = [ { frequency = "monthly" } ]
+recurrence_rules = [ { frequency = "weekly" } ]
+
+# Past recurring events
 
 [[events]]
-uid = "2be23d34f40821a2793cf8bf2db5b1bcbfe620f3"
-title = "Lang Team Design Meeting"
-description = "https://github.com/orgs/rust-lang/projects/31/views/10"
+uid = "9fa5f574898e6a2417d9ac9b20b5294507c15e2a"
+title = "Lang Team Planning Meeting"
+description = "In this meeting, we plan the design meetings for the month."
 location = "https://meet.jit.si/ferris-rules"
-last_modified_on = "2024-02-29T00:00:00.00Z"
+last_modified_on = "2024-10-22T00:00:00.00Z"
 start = { date = "2024-03-06T12:30:00.00", timezone = "America/New_York" }
 end = { date = "2024-03-06T14:00:00.00", timezone = "America/New_York" }
 status = "confirmed"
 transparency = "opaque"
 organizer = { name = "t-lang", email = "lang@rust-lang.org" }
-recurrence_rules = [ { frequency = "weekly" } ]
-
-# Past recurring events
```

Comment:
Note that the way this diff is rendered is unhelpful.  What actually happened is this event was moved down to the "Past recurring events" block and its recurrence was altered to add an end datetime.